FXyz-Core/build.gradle

```diff
@@ -7,5 +7,11 @@ repositories {
 dependencies {
     api ('eu.mihosoft.vrl.jcsg:jcsg:0.5.7') { transitive = false }
     api 'eu.mihosoft.vvecmath:vvecmath:0.4.0'
-    api 'org.orbisgis:poly2tri-core:0.1.2'
-}
\ No newline at end of file
+    api ('org.orbisgis:poly2tri-core:0.1.2') {
+        exclude group: 'org.slf4j', module: 'slf4j-api'
```

Comment:
Since you are removing this api dependency from FXyz-Core, FXyz-Imports, that depends on FXyz-Core fails to find it.
So it can be fixed adding the same dependency as implementation for FXyz-Imports. Does that make sense to you?

Reply:
Sorry about that, I was testing samples jlink and forgot to run tests on this one haha.

I just realized importers had `api (project(':FXyz-Core')) { transitive = false }` transitives set to false.

I updated it so that the runtimeElements have the dependencies, hence the second `implementation (project(':FXyz-Core'))`

This keeps apiElements metadata correct. So there's no changes downstream.

Reply:
> So it can be fixed adding the same dependency as implementation for FXyz-Imports. Does that make sense to you?

Sadly it would've failed since transitives is set to false. I would've needed to change module-info.java to use `requires static org.slf4j` to make that work but then jlink would not be happy since slf4j would not an automatic module anymore.